app/controllers/api/v1/keywords_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
 
       def create
         if csv_form.save(create_params[:file])
+          Google::DistributeSearchJob.perform_later(csv_form.keyword_ids)
```

Comment:
As this PR is for web backend, this change should be in the API task.

Reply:
Roger that, I'll create a new issue, move this change into it with a new PR. 
Well identified @junan ~